### PR TITLE
Make strWalletFile const

### PR DESCRIPTION
**Pull request by spencerlievens**

### CODE REVIEW

**Comment by spencerlievens** [comment]
Unsure why this fails on walletbackup test. Anyone?

File: src/wallet/wallet.h [hunk 1]
```diff
@@ -579,11 +579,10 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
         SetNull();
     }
 
-    CWallet(const std::string& strWalletFileIn)
+    CWallet(const std::string& strWalletFileIn) : strWalletFile(strWalletFileIn)
```
Comment (tgflynn) [inline src/wallet/wallet.h]: I prefer to see class initializers on a second line.

Comment (spencerlievens) [inline src/wallet/wallet.h]: I can make that change, thats not an issue ;)

However, any ideas on the test failing?

Comment (spencerlievens) [inline src/wallet/wallet.h]: Also, are you sure you want that on a new line? Take a look at wallet.h as a whole.

@tgflynn

Comment (tgflynn) [inline src/wallet/wallet.h]: Yes, I believe it's easier to read if the initialization is separated from the constructor prototype.

> Take a look at wallet.h as a whole.

Not sure what you're referring to.

RE: Travis. There seems to be quite a bit of non-determinism in the travis tests.  It's something we're looking into but we don't have a solution for that problem yet.

**Comment by spencerlievens** [comment]
@tgflynn I have done that, now let's see if Travis-CI passes this time around.

**Pull request merged**